src/Sarif.Converters/PyLintConverter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Microsoft.CodeAnalysis.Sarif.Converters.PyLintObjectModel;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters
+{
+    public class PyLintConverter : ToolFileConverterBase
```

Comment:
>PyLintConverter [](start = 17, length = 15)

Very sorry to point this out so late, but you've got the casing for Pylint wrong. See external web sites for proper casing. You'll need to change this everywhere, also correct file names. Don't forget VS can rename things for you. :) In this case, actually, a global replace across files using exact casing + whole word options will get you most of the way there in an easy way.